spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -134,6 +134,18 @@
 	 */
 	public static final String SPAN_PEER_SERVICE_TAG_NAME = "peer.service";
 
+	/**
+	 * ID of the instance from which the span was originated. This tag is related to the
+	 * client side communication of an RPC call.
+	 */
+	public static final String SPAN_CLIENT_INSTANCEID = "client.instanceid";
```

Comment:
I see one issue with this (but maybe actually I'm wrong then there's no issue at all ;) ). If both the client and the server put the same tag with 2 different values, won't it be broken on the Zipkin UI in the RPC span?